### DISCUSSION TITLE
Creating EventEnrollments

### DISCUSSION THREAD

**Discussion by juraj3579** [JoelBender/bacpypes#410] [Q&A] [answered]
Hi everyone,

is it possible to create objects on other devices with bacpypes?

I need to create bunch of eventenrollments on Tridium Jace 8000. There is no possibility, to create these object locally on the station. Therefor, I'm searching for options to create them from "outside", however, It's all quite a blind fly.

**Comment by JoelBender** [comment, marked as answer]
You are looking for clause **15.3 CreateObject Service** which has three protocol pieces, **CreateObjectRequest** goes out to the server which responds with a **CreateObjectAck** and provides the object identifier of the new object, or a **CreateObjectError** with a reference to the first failed property/value that's a problem (invalid datatype, value out of range, etc).  Missing required properties _might_ be filled in with reasonable defaults, but that's up to the server.

**Reply by juraj3579** [reply]
Thank you for your help. 
I've managed to put working app together. Unfortunately, got reject with (abort - other 56) to the request. 
I also tried to create an object with BACEye, which worked. However, see no difference between these two requests, the one from BACEye and second using BACPypes, when comparing them with Wireshark. So I have no idea, what causes the error and reject.
I think it's more the issue with the controller, but any help or suggestions would be appreciate, if possible.

EDIT: It's working now. Just changed ObjectID of the test object i've tried to create. If the CreateObjectRequest fails on the device (prop error, etc.), the object is still created on the controller, but not visible on BACnet Network, so I can't use the same ObjectID. However, thank you once more for your help.
